examples/scripts/wot-servient.conf.json

```diff
@@ -28,6 +28,18 @@
 			"priority": 10
 		}
 	},
+    "wotfirestore": {
```

Comment:
So this is where you describe the configuration for Tutorial.
I thought it was a place to put the JSON format of the configuration.
I'll delete it.

Reply:
So it should be deleted from here (in my opinion) but it can and should be put at the package's README and also https://github.com/eclipse/thingweb.node-wot/blob/5ffd659a2502657151ad4927dacab3849a09a060/packages/cli/src/cli.ts#L220